docs/manuals/core/entities/task-mgt.md

```diff
@@ -78,7 +82,22 @@ task_2 - pipeline.predicting
 # task_1 == task_2
 ```
 
-All the jobs can be retrieved using the method `taipy.get_tasks()^`.
+Tasks can also be retrieved by their config_id by calling `taipy.get_entities_by_config_id()^` with their config_id.
```

Comment:
"by their config_id by calling \`taipy.get_entities_by_config_id()^\` with their config_id" => "using the method  \`taipy.get_entities_by_config_id()^\`, providing the configuration identifier"